entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+# Drop root privileges if we are running gemstash as root.
+if [ "$1" = "gemstash" ] && [ "$(id -u)" = "0" ]; then
+    # Change the ownership of user-mutable directories to gemstash
+    chown -R "${GEMSTASH_USER}:${GEMSTASH_USER}" "${GEMSTASH_HOME}/data"
+
+    # Run gemstash as gemstash user
+    command="tini -- ${*} --config-file=${GEMSTASH_HOME}/app/config.yml.erb"
```

Comment:
`${*}` is equivalent to `${@}` which is pretty much the command that's passed in. In out case, that would be `gemstart start --no-daemonize`. So here we're appending `--config-file` to make sure we are using our ERB config and running that command with `tini`. This is then passed to `su-exec` to run as `gemstash`.